### PR TITLE
Theta Project Changes v1.2

### PR DESCRIPTION
**Pull request by lngredients**
Changed Theta Project Armor to a cooler looking armor set.
Theta armor should be capable of holding most weapons in its suit slot.
Gave the theta project closet a bullpup rifle and three mags to equip their creation, or be killed by it.
The armor gives a slight boost to walk speed.
Helmet light is twice as strong as a normal helmet light so you can actually see in the dark.
Theta Undersuit was replaced and now will actually work.
Slightly Changed Theta Project equipment names so they arn't so long.
Decreased armor on boots and gloves because the full body suit is strong enough.

### CODE REVIEW

**Comment by lngredients** [comment]
Octopus it looks to me like the alpha locker and freezer are conflicting for some reason I don't understand.

**Comment by lngredients** [comment]
I have no idea what is wrong. It CI says it's missing HUD elements and other things like that which I didn't mess with.

**Comment by lngredients** [comment]
Does anyone know what I can do to make this playable?

File: baystation12.dme [hunk 1]
```diff
@@ -2617,10 +2617,9 @@
 #include "maps\crashsite_zeta\crashsite_zeta_map.dm"
 #include "maps\desert_outpost\__desert_outpost_map.dm"
 #include "maps\example\example_define.dm"
-#include "maps\first_contact\all_includes.dm"
-#include "maps\first_contact\mapdef.dm"
+#include "maps\geminus_city\geminus_city_jobs.dm"
```
Comment (bloxgate) [inline baystation12.dme]: This here is likely the problem. You seem to be including some of the map, but not all the files that are required for it to work. If you're trying to change the map to geminus, you'll need to include maps\geminus_city\geminus_city.dm

**Review by bloxgate — CHANGES_REQUESTED** [review]

**Comment by lngredients** [comment]
So adding in the requested DM  maps\geminus_city\geminus_city.dm, and I am still receiving the same errors as before it was included. 

This is the CI information the geminus-city.dm included.
https://travis-ci.org/HaloSpaceStation/HaloSpaceStation13/builds/386630838

I added in the geminuscity_2.dmm to test and see results. I will link that CI build below.
https://travis-ci.org/HaloSpaceStation/HaloSpaceStation13/builds/386635391

**Pull request closed without merging**